x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/findings_by_resource_container.tsx

```diff
@@ -49,11 +50,19 @@ export const FindingsByResourceContainer = ({ dataView }: FindingsBaseProps) =>
     <Route
       exact
       path={findingsNavigation.findings_by_resource.path}
-      render={() => <LatestFindingsByResource dataView={dataView} />}
+      render={() => (
+        <TrackApplicationView viewId={findingsNavigation.findings_by_resource.id}>
+          <LatestFindingsByResource dataView={dataView} />
+        </TrackApplicationView>
```

Comment:
we could remove the inner routes, and combine `findingsNavigation` with `cloudPosturePages`, like we did for benchmarks/rules. 

this way we could remove the above code and the initial root-routes wrapper would work. 

not critical for now imo tho